src/unitxt/metrics.py

```diff
@@ -2108,7 +2108,17 @@ class F1MacroMultiLabel(F1MultiLabel):
     average = None
 
 
-class Rouge(InstanceMetric):
+class NLTKMixin(Artifact):
+    def prepare(self):
+        super().prepare()
+        import nltk
```

Comment:
make add as a _requirement_list?

Reply:
There is not way to merge the requirements list of both classes